install_ffmpeg.sh

```diff
@@ -63,11 +63,11 @@ fi
 if [ ! -e "$ROOT/x264" ]; then
   git clone http://git.videolan.org/git/x264.git "$ROOT/x264"
   cd "$ROOT/x264"
-  if [[ ! $IS_M1 ]]; then
+  if [[ $IS_M1 ]]; then
```

Comment:
IIUC prior to this change we would checkout the newer version of x264 if the user is *not* running on M1 . So, I changed this conditional so we checkout the newer version of x264 if the user *is* running on M1.